packages/js/test/plugins/auctionHouseModule/withdrawFromFeeAccount.test.ts

```diff
@@ -0,0 +1,73 @@
+import test, { Test } from 'tape';
+import { sol, toPublicKey } from '@/types';
+import { metaplex, killStuckProcess, createWallet } from '../../helpers';
+
+killStuckProcess();
+
+test('[auctionHouseModule] withdraw from fee account on an Auction House', async (t: Test) => {
+  // Given we have an Auction House and airdropped 100 SOL to fee account.
+  const mx = await metaplex();
+
+  const { auctionHouse } = await mx
+    .auctionHouse()
+    .create({
+      sellerFeeBasisPoints: 200,
+    })
+    .run();
+
+  await mx.rpc().airdrop(auctionHouse.feeAccountAddress, sol(100));
+
+  // When we withdraw 1 SOL from fee account.
+  await mx
+    .auctionHouse()
+    .withdrawFromFeeAccount({
+      auctionHouse,
+      amount: sol(1),
+    })
+    .run();
+
+  // Then fee account has 99 SOL in it.
```

Comment:
Can we also check that the destination has one more SOL? I know we check this on the other test but it would be nice to have one test that checks both the sending and the receiving part to be safe.